### DISCUSSION TITLE
why it is infinity loading on `/app/[slug]` but not `/app/slug/[slug]`?

### DISCUSSION THREAD

**Discussion by dotku** [vercel/next.js#67199] [App Router] [answered]
Can anyone knows why?
Here is my code

```
export default function Page({ params }) {
  return <div>My Post 003: {params.slug.slice(3)}</div>;
}
```

**Comment by icyJoseph** [comment]
Got a repository to take a look at?

**Reply by dotku** [reply]
Here is the PR with sample: https://github.com/dotku/next-appointment-app/pull/22/files

**Reply by icyJoseph** [reply, marked as answer]
Well, I can't reproduce... it works fine with this tree:

 ```console
src/app
├── [slug]
│   └── page.tsx
├── about
│   └── page.tsx
├── layout.tsx
├── page.tsx
└── slug
    └── [slug]
        └── page.tsx
```

No problem at all. I can navigate to `http://localhost:3001/slug/foo`, or `http://localhost:3001/foo`, and it works.

Could you try to make a separate repository with as few changes as possible, where you reproduce this?

**Reply by dotku** [reply]
It is working now after I rebboot my laptop... not sure what cause the infinity loading issue; will keep eyes on it. Thanks :)
